src/antelope/stores/rex.ts

```diff
@@ -198,9 +292,11 @@ export const useRexStore = defineStore(store_name, {
             (this.__rexData[label] as EvmRexModel).withdrawable = withdrawable;
         },
         setDeposits(label: string, deposits: EvmRexDeposit[]) {
-            this.trace('setDeposits', label, ... deposits.map(d => parseFloat(ethers.utils.formatUnits(d.amount, this.getStakingDecimals()))));
+            const copia = deposits.slice();
```

Comment:
It was needed because the parameter 'deposits' is a read-only variable and I can't sort it before storing it.
I changed the variable name and added a comment to explain this fact.